shared/threat-models/ext/threat-model-grouping.model.yml

```diff
@@ -21,3 +21,6 @@ extensions:
       # Android threat models
       - ["android-external-storage-dir", "android"]
       - ["contentprovider", "android"]
+
+      # Threat models that are not grouped with any other threat models
+      #   "reverse-dns"
```

Comment:
This is probably fine the way it is, though it could be confusing if we start having lots of un-rooted threat models. 

The reason this works is that we want the ability for people to define their own threat models (and even threat model groupings) that are neither `local` nor `remote`.

Not for this PR, but I wonder if we should refactor so that there is an `all` threat model that can be used to include all threat models defined in the standard libraries (though I'm not sure `all` may already be special threat model that is hard coded, so we could bike shed a different name).

Reply:
There already exist a hardcoded `all` threat model that enables "everything" also threat models that are not listed in any groups.

Reply:
I think we should strive for documenting all the threat models that we supply as a part of our external documentation. It is also a good idea to list them in the shared file as is done in this PR.